### ISSUE TITLE
 Unable to extend undefined GraphQL union

### ISSUE THREAD

**Issue by veeramarni** [ardatan/graphql-toolkit#324]
When having `union` type in one schema and `extend union` of same type in a different file, we get the following error. 

Schema1
```

  type Query {
    fooBar: [FooBar!]!
  }

  union FooBar = Foo | Bar

  type Foo {
    id: ID!
  }

  type Bar {
    id: ID!
  }

```

Schema2

```
  extend union FooBar = Zoo

  type Zoo {
    id: ID!
  }
```
We use `graphql-code-generator` for generating types and it fails with below error. If we put the `extend union` into the same file as `union`
```
Failed to load schema from ./packages/graphql/**/*.graphql:

        Unable to extend undefined GraphQL union: [object Object]
        Error: Unable to extend undefined GraphQL union: [object Object]
```

**Comment by veeramarni** [comment]
@ardatan  I have tested with commenting those lines and it works fine. How soon this can be published?

**Comment by ardatan** [comment]
@veeramarni I will publish it tomorrow :)

**Comment by ardatan** [comment]
Fixed in 0.7.0!
